src/main/java/org/mule/extension/db/internal/domain/connection/oracle/OracleConnectionUtils.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.extension.db.internal.domain.connection.oracle;
+
+/**
+ * Utils for the oracle db connection
+ *
+ */
+public class OracleConnectionUtils {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1118))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-core-extensions-mule-db-connector&pullRequest=156&issues=AWkWM61JotIXnmsUtJ_Z&open=AWkWM61JotIXnmsUtJ_Z)

[ISSUE_KEY]: AWkWM61JotIXnmsUtJ_Z